astropy/io/fits/util.py

```diff
@@ -42,6 +42,8 @@ class StringIO(object):
 elif six.PY2:
```

Comment:
Not related to this PR, but I just notice that this logic here is not compatible with Python 4.